src/flaskwebgui.py

```diff
@@ -271,10 +283,12 @@ def start_browser(self, server_process: Union[Thread, Process]):
         if isinstance(server_process, Process):
             if self.on_shutdown is not None:
                 self.on_shutdown()
+            cleanup_temp_dirs()
             server_process.kill()
         else:
             if self.on_shutdown is not None:
                 self.on_shutdown()
+            cleanup_temp_dirs()
```

Comment:
Same here:
Replace `cleanup_temp_dirs()` func with `shutil.rmtree(self.profile_dir, ignore_errors=True)`

Make sure to bump version in setup.py file